README.md

```diff
@@ -33,6 +33,21 @@ static $result_count_max = 10;
 
 This would set to a maximum of ten (10) results.
 
+## Meeting Search Radius ##
+
+Change the default meeting search radius, this can be in miles or an integer (auto radius). You can change this in your `config.php` with the following:
+
+```php
+static $meeting_search_radius = 30;
+```
+This would set the radius to a maximum of 30 miles.
+
+```php
+static $meeting_search_radius = -50;
+```
+This would set the radius to a integer of -50 and is the default.
```

Comment:
If the number is negative it means the first n results.  In this case the first 50 regardless of distance.